hack/run-ci-e2e-test.sh

```diff
@@ -3,6 +3,9 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
+# Set up hybrid networking on the cluster, a requirement for Windows support in OpenShift
```

Comment:
Do you want to create a TODO for this or waiting for this to be removed?

Reply:
> . Though from what I understand this will just be a no-op given that it will result in no changes to the object.

This is what I meant by waiting to be removed. No need for this at all